common/src/main/java/org/apache/rocketmq/common/message/MessageClientIDSetter.java

```diff
@@ -107,6 +104,15 @@ public static String getIPStrFromID(String msgID) {
         return result;
     }
 
+    public static short getPidFromID(String msgID) {
```

Comment:
it seems this method repeat with org.apache.rocketmq.common.message.MessageClientIDSetter#getIPFromID